recipe/build_base.sh

```diff
@@ -391,7 +393,7 @@ fi
 ln -s ${PREFIX}/bin/python${VER} ${PREFIX}/bin/python
 ln -s ${PREFIX}/bin/pydoc${VER} ${PREFIX}/bin/pydoc
 # Workaround for https://github.com/conda/conda/issues/10969
-ln -s ${PREFIX}/bin/python3.10 ${PREFIX}/bin/python3.1
+ln -s ${PREFIX}/bin/python3.11 ${PREFIX}/bin/python3.1
```

Comment:
Should it be `ln -s ${PREFIX}/bin/python3.11 ${PREFIX}/bin/python3.11`? How is the folder really named if we take into account the conda issue #46  ?

Reply:
Yes this is correct, it will truncate 3.11 into 3.1 in the same way. See root cause [here](https://github.com/conda/conda/blob/33a142c16530fcdada6c377486f1c1a385738a96/conda/common/path.py#L181)